.gitignore

```diff
@@ -21,3 +21,6601 @@ captures
 acra.properties
 respect-lib-shared/src/commonMain/composeResources/files/buildinfo.properties
 respect-server/log/
+/cypress/downloads/000_000_hello_world.mp4
```

Comment:
We should **NOT** have an extra 6,000 lines of gitignore. 

node_modules and downloads should be added to gitignore as a folder not by listing every file.